packages/web-components/src/components/cta-block/cta-block.ts

```diff
@@ -122,7 +125,7 @@ class C4DCTABlock extends StableSelectorMixin(C4DContentBlock) {
     });
     return html`
       <div ?hidden="${!_hasContent}" class="${classes}">
```

Comment:
Should put a `part` attribute here for completeness
```suggestion
      <div ?hidden="${!_hasContent}" class="${classes}" part="helper-wrapper">
```